test/functionalTest/cases/0947_v2_list_all_entities/GET_v2_entities.test

```diff
@@ -112,14 +112,12 @@ echo
 echo "05. POST /v2/entities with compound attribute value"
 echo "==================================================="
 payload='{
-      "id":   "E2",
-      "type": "T1",
-      "A0" : 
-        {
-          "value" : { "i1": "1", "s1": "s" }
-        }
-      
-
+  "id":   "E2",
```

Comment:
I found a few functests that had really bad indentations, among the failing ones.
Now that I was there, I fixed that also ...

The only 'real' changes in all these functests are to add 'metadata: { *** }'  and 'type: null'